scripts/zkai_attention_derived_d128_native_mlp_proof_route_gate.py

```
@@ -992,8 +1340,13 @@ def validate_payload(payload: Any, *, context: dict[str, Any] | None = None) ->
         MISSING_MATCHED_SEPARATE_ENVELOPES
     ):
         raise NativeMlpProofRouteError("matched separate baseline boundary drift")
+    if (
+        data["comparison"].get("matched_six_separate_derived_baseline_status")
+        != "COMPLETE_EXACT_SIX_DERIVED_SEPARATE_ENVELOPES"
+    ):
+        raise NativeMlpProofRouteError("matched six-separate baseline status drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)